lib/engine/game/g_1849/round/operating.rb

```diff
@@ -7,11 +7,24 @@ module Game
     module G1849
       module Round
         class Operating < Engine::Round::Operating
+          attr_reader :issued_bond, :redeemed_bond
+
+          def setup
+            @issued_bond = {}
+            @redeemed_bond = {}
+
+            super
+          end
+
           def next_entity!
             return @game.end_game! if @entities[@entity_index].reached_max_value
 
             super
           end
+
+          def cash_crisis_entity
+            @game.corporations.find { |corp| corp.cash.negative? }
+          end
```

Comment:
None of the changes in this file should be needed. The @issue_bond and @redeemed_bound should be done via `round_state` and `cash_crisis_entity` can be done directly in the emr step.

Reply:
done